### ISSUE TITLE
handshapes not entered properly into the utterance window from the morph-phon window

### ISSUE THREAD

**Issue by justberg** [DCS-LCSR/SignStream3#259]
HI - This is Carol here with Justin.  We have a file in which we have entered the letters for the fingerspelled sign "dog" - D-O-G.  These appear correctly in the morph-phon window, but get converted to D-O-O when entered into the utterance window!

Here are 3 screen shots showing the sequential order.
I also used your latest test version of the program, so am attaching the log and error files, in case that is revealing.

THanks!

<img width="273" alt="Screen Shot 2019-06-10 at 10 33 40 AM" src="https://user-images.githubusercontent.com/51125212/59202881-6db74680-8b6b-11e9-9bc6-97cb3c7fe05c.png">

<img width="545" alt="Screen Shot 2019-06-10 at 10 33 54 AM" src="https://user-images.githubusercontent.com/51125212/59202894-73ad2780-8b6b-11e9-9415-dedd1cc861fb.png">

<img width="359" alt="Screen Shot 2019-06-10 at 10 34 03 AM" src="https://user-images.githubusercontent.com/51125212/59202906-7d368f80-8b6b-11e9-8384-fde2cc3ac4d5.png">

[SignStreamErr.txt](https://github.com/DCS-LCSR/SignStream3/files/3272449/SignStreamErr.txt)
[SignStreamLog.txt](https://github.com/DCS-LCSR/SignStream3/files/3272450/SignStreamLog.txt)

**Comment by cneidle** [comment]
Really?  This reproduces very reliably for me!  Have you tried entering this sequence of letters?  I suspect it is something wrong with the way the "G" handshape is represented. I went to a random file, added fs-DOG and exactly the same thing happened.  When I went back to the morph-phon window, the "G" was also wrong there, although it was initially right.

<img width="1408" alt="Screen Shot 2019-06-10 at 4 14 34 PM" src="https://user-images.githubusercontent.com/13629122/59223918-1d57dd00-8b9b-11e9-83e5-e0aa50faf203.png">
